src/Sulu/Bundle/ContentBundle/Resources/public/js/components/single-internal-link/main.js

```diff
@@ -185,46 +184,27 @@ define([], function() {
         /**
          * initialize column navigation
          */
-        overlayOkCallback = function () {
```

Comment:
but it is quite strange when you select a single item and have to click on "OK"? @berndhep what do you think?